### PR TITLE
welcome message when installing packages

### PR DESCRIPTION
**Pull request by sabban**
add welcome message when installing packages

### CODE REVIEW

File: debian/postinst [hunk 1]
```diff
@@ -102,6 +102,13 @@ if [ "$1" = configure ]; then
         echo "Not attempting to start crowdsec, port ${PORT} is already used or lapi was disabled"
         echo "This port is configured through /etc/crowdsec/config.yaml and /etc/crowdsec/local_api_credentials.yaml"
     fi
+
+    echo "Get started with CrowdSec:"
+    echo " * detailed guides are available in our documentation: https://docs.crowdsec.net"
+    echo " * community-driven configuration items are available at the Hub: https://hub.crowdsec.net"
+    echo " * Get insight of your CrowdSec usage thanks to the console: https://app.crowdsec.net"
```
Comment (LaurenceJJones) [inline debian/postinst:109]: Gain insights into your use of CrowdSec with the help of the console

File: rpm/SPECS/crowdsec.spec [hunk 2]
```diff
@@ -179,6 +179,11 @@ if [ $1 == 1 ]; then
     cscli hub update
     CSCLI_BIN_INSTALLED="/usr/bin/cscli" SILENT=true install_collection
 
+    echo "Get started with CrowdSec:"
+    echo " * detailed guides are available in our documentation: https://docs.crowdsec.net"
+    echo " * community-driven configuration items are available at the Hub: https://hub.crowdsec.net"
+    echo " * Get insight of your CrowdSec usage thanks to the console: https://app.crowdsec.net"
```
Comment (LaurenceJJones) [inline rpm/SPECS/crowdsec.spec:185]: Gain insights into your use of CrowdSec with the help of the console

File: rpm/SPECS/crowdsec.spec [hunk 3]
```diff
@@ -179,6 +179,11 @@ if [ $1 == 1 ]; then
     cscli hub update
     CSCLI_BIN_INSTALLED="/usr/bin/cscli" SILENT=true install_collection
 
+    echo "Get started with CrowdSec:"
+    echo " * detailed guides are available in our documentation: https://docs.crowdsec.net"
+    echo " * community-driven configuration items are available at the Hub: https://hub.crowdsec.net"
```
Comment (LaurenceJJones) [inline rpm/SPECS/crowdsec.spec:184]: Configuration items created by the community can be found at the Hub

File: debian/postinst [hunk 4]
```diff
@@ -102,6 +102,13 @@ if [ "$1" = configure ]; then
         echo "Not attempting to start crowdsec, port ${PORT} is already used or lapi was disabled"
         echo "This port is configured through /etc/crowdsec/config.yaml and /etc/crowdsec/local_api_credentials.yaml"
     fi
+
+    echo "Get started with CrowdSec:"
+    echo " * detailed guides are available in our documentation: https://docs.crowdsec.net"
+    echo " * community-driven configuration items are available at the Hub: https://hub.crowdsec.net"
```
Comment (LaurenceJJones) [inline debian/postinst:108]: Configuration items created by the community can be found at the Hub

**Review by mmetc — APPROVED** [review]

**Pull request merged**